### ISSUE TITLE
Export type `LucideIcon`?

### ISSUE THREAD

**Issue by jimmiejackson414** [swisnl/nuxt-lucide-icons#13]
I have several instances in my app where I'd imported the `LucideIcon` type from the `lucide-vue-next` library to define a prop type for example, however from what I can tell this library has no such export. Is that right, or am I totally missing it?

**Comment by JaZo** [comment]
Correct, this library doesn't have such a type. What would be the advantage of providing one, as it's already available in `lucide-vue-next`?
